src/Payments.sol

```diff
@@ -425,6 +469,17 @@ contract Payments is
         );
     }
 
+    /// @notice Modifies the payment rate and optionally makes a one-time payment. The rail will automatically be settled at the current rate (before changing it) unless the rail is subject to arbitration.
+    /// - If the rail has already been terminated for an entire lockup period, this method will always revert.
```

Comment:
See https://github.com/FilOzone/payments/issues/16

Reply:
note that we're also moving to enqueuing the rate change and NEVER settling in this method.